src/components/search/data/reducer.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable import/prefer-default-export */
+import {
+  DELETE_KEY, SET_KEY, CLEAR_FILTERS, SET_MULTIPLE_KEYS, SET_ARRAY_VALUE, REMOVE_ARRAY_VALUE,
```

Comment:
Change to SET_ARRAY_VALUE to something that has REFINEMENT in it's name. :)
Same for REMOVE_ARRAY_VALUE